stable/datadog/templates/container-trace-agent.yaml

```diff
@@ -17,6 +17,13 @@
     protocol: TCP
   env:
     {{- include "containers-common-env" . | nindent 4 }}
+    {{- if .Values.datadog.clusterName }}
+    {{- if not (regexMatch "^([a-z]([a-z0-9\\-]{0,38}[a-z0-9])?\\.)*([a-z]([a-z0-9\\-]{0,38}[a-z0-9])?)$" .Values.datadog.clusterName) }}
+    {{- fail "Your `clusterName` isn’t valid. It must be dot-separated tokens where a token start with a lowercase letter followed by up to 39 lowercase letters, numbers, or hyphens and cannot end with a hyphen."}}
+    {{- end}}
+    - name: DD_CLUSTER_NAME
+      value: {{ .Values.datadog.clusterName | quote }}
+    {{- end }}
```

Comment:
can we put this in the "common env var" ?
I feel like it can be also used by the process-agent

Reply:
I moved it to `containers-common-env` as well as the `DOCKER_HOST` as they are both needed to compute the hostname by the agent.